jrnl/jrnl.py

```diff
@@ -77,6 +78,10 @@ def _is_write_mode(args, config, **kwargs):
             args.edit,
             args.export,
             args.end_date,
+            args.reminisce,
```

Comment:
Working on a new name for this arg (see the issue).

Reply:
👍

Reply:
Done.